### PR TITLE
No issue: adding E2E missing javadoc

### PR DESCRIPTION
**Pull request by victoralfaro-dotcms**

### CODE REVIEW

**Review by jcastro-dotcms — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

**Review by freddyDOTCMS — APPROVED** [review]

File: .github/filters.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ backend: &backend
   - 'core-web/pom.xml'
   - 'dotCMS/src/main/webapp/html/**/!(*.{css,js})'
   - 'dotcms-postman/**'
+  - 'dotcms-e2e/**''
```
Comment (spbolton) [inline .github/filters.yaml:15]: Typo here extra quote

Comment (victoralfaro-dotcms) [reply .github/filters.yaml]: Fixed.

**Comment by spbolton** [comment]
These issues were only found on the merge queue as e2e changes were not triggering running in PR. These should be detected and fixed

https://github.com/dotCMS/core/actions/runs/10944207433/job/30385538620#step:6:3438
https://github.com/dotCMS/core/actions/runs/10944207433/job/30385538620#step:6:3438

File: .github/filters.yaml [hunk 2]
```diff
@@ -12,6 +12,7 @@ backend: &backend
   - 'core-web/pom.xml'
   - 'dotCMS/src/main/webapp/html/**/!(*.{css,js})'
   - 'dotcms-postman/**'
+  - 'dotcms-e2e/**'
```
Comment (spbolton) [inline .github/filters.yaml:15]: The folder here is to match is e2e and not dotcms-e2e.

**Review by dcolina — APPROVED** [review]

**Review by nollymar — APPROVED** [review]

**Pull request merged**